packages/components/psammead-most-read/src/Rank/index.jsx

```diff
@@ -132,9 +141,7 @@ const MultiColumnWrapper = styled(TwoColumnWrapper)`
   /* 5 columns of items at viewport 1280px and above */
   @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
     min-width: ${props =>
-      listHasDoubleDigits(props.numberOfItems)
-        ? isFiveOrTen(props)
-        : getRankMinWidth(props).group5};
+      listHasDoubleDigits(props.numberOfItems) ? isFiveOrTen(props) : 'auto'};
```

Comment:
I update this to `auto` as when there is no double-digit (i.e numbers 1-5) on the 5th group, the numbers will be always be aligned side by side and never on top of each other so there is no need to set a min-width for alignment. 

![image](https://user-images.githubusercontent.com/30599794/77918670-1e760a00-7294-11ea-825a-994c84f75389.png)